.rspec

```diff
@@ -1,2 +1,3 @@
 --format documentation
 --color
+--require spec_helper
```

Comment:
Forgot we had .rspec as part of the gem - I like this change a lot (and the subsequent deletes in the rspec files.)  🍕